src/OpenTelemetry/Internal/EnvironmentVariableHelper.cs

```diff
@@ -111,5 +111,33 @@ public static bool LoadUri(string envVarKey, out Uri result)
 
             return true;
         }
+
+        /// <summary>
+        /// Reads an environment variable and parses it as a <see cref="bool" />.
+        /// </summary>
+        /// <param name="envVarKey">The name of the environment variable.</param>
+        /// <param name="result">The parsed value of the environment variable.</param>
+        /// <returns>
+        /// Returns <c>true</c> when a non-empty value was read; otherwise, <c>false</c>.
+        /// </returns>
+        /// <exception cref="FormatException">
+        /// Thrown when failed to parse the non-empty value.
+        /// </exception>
+        public static bool LoadBoolean(string envVarKey, out bool result)
+        {
+            result = default;
+
+            if (!LoadString(envVarKey, out string value))
+            {
+                return false;
+            }
+
+            if (!bool.TryParse(value, out result))
```

Comment:
I would rather handle only `true` and `false` - case sensitive - just like other enum values like `OTEL_EXPORTER_JAEGER_PROTOCOL`. I like consistency 😉

Reply:
The advantage of starting with a minimal subset of supported values is that we won't make a little breaking change if we later decide that this should be case-sensitive. Personally, I think having "strict" requirements for values may even occur more maintainable for the users, because it makes their configuration consistent.

PS. It is not a strong opinion 😉